itu-minitwit/minitwit_tests.py

```diff
@@ -58,46 +58,46 @@ def add_message(self, text):
         rv = self.app.post('/add_message', data={'text': text},
                                     follow_redirects=True)
         if text:
-            assert 'Your message was recorded' in rv.data
+            assert b'Your message was recorded' in rv.data
         return rv
 
     # testing functions
 
     def test_register(self):
         """Make sure registering works"""
         rv = self.register('user1', 'default')
-        assert 'You were successfully registered ' \
-               'and can login now' in rv.data
+        assert b'You were successfully registered ' \
```

Comment:
@licraux  thank you, I checked your changes in this file and refactored using that so the test could pass :)